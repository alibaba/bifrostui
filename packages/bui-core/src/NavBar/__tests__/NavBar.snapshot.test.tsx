import { snapshotTest } from 'testing';

describe('NavBar demo snapshot', () => {
  snapshotTest('NavBar');
});
