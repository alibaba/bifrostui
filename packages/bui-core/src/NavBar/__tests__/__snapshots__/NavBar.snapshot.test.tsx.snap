// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar demo snapshot NavBar demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    />
    <div
      className="bui-navbar-right"
    />
  </div>
</div>
`;

exports[`NavBar demo snapshot NavBar demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "50px",
    }
  }
>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"m48 43.7 25.8-25.8c1.2-1.2 3.1-1.2 4.3 0 1.2 1.2 1.2 3.1 0 4.3L52.3 48l25.8 25.8c1.2 1.2 1.2 3.1 0 4.3s-3.1 1.2-4.3 0L48 52.3 22.2 78.1c-1.2 1.2-3.1 1.2-4.3 0-1.2-1.2-1.2-3.1 0-4.3L43.7 48 17.9 22.2c-1.2-1.2-1.2-3.1 0-4.3 1.2-1.2 3.1-1.2 4.3 0L48 43.7z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    >
      标题文案
    </div>
    <div
      className="bui-navbar-right"
    />
  </div>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    />
    <div
      className="bui-navbar-title ellipsis"
    >
      标题文案
    </div>
    <div
      className="bui-navbar-right"
    />
  </div>
</div>
`;

exports[`NavBar demo snapshot NavBar demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        color="#ff335c"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    >
      标题文案
    </div>
    <div
      className="bui-navbar-right"
    />
  </div>
</div>
`;

exports[`NavBar demo snapshot NavBar demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "50px",
    }
  }
>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
      返回
    </div>
    <div
      className="bui-navbar-title ellipsis"
    />
    <div
      className="bui-navbar-right"
    />
  </div>
</div>
`;

exports[`NavBar demo snapshot NavBar demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "50px",
    }
  }
>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    >
      标题文案
    </div>
    <div
      className="bui-navbar-right"
    />
  </div>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    >
      这个标题文案很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长很长
    </div>
    <div
      className="bui-navbar-right"
    />
  </div>
</div>
`;

exports[`NavBar demo snapshot NavBar demo snapshot 0 6`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "50px",
    }
  }
>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    >
      标题文案
    </div>
    <div
      className="bui-navbar-right"
    >
      文案
    </div>
  </div>
  <div
    className="bui-navbar"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    >
      标题文案
    </div>
    <div
      className="bui-navbar-right"
    >
      <svg
        className="bui-svg-icon icon-size-large"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M22 40c4.4 0 8 3.6 8 8s-3.6 8-8 8-8-3.6-8-8 3.6-8 8-8zm26 0c4.4 0 8 3.6 8 8s-3.6 8-8 8-8-3.6-8-8 3.6-8 8-8zm26 0c4.4 0 8 3.6 8 8s-3.6 8-8 8-8-3.6-8-8 3.6-8 8-8z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
  </div>
</div>
`;

exports[`NavBar demo snapshot NavBar demo snapshot 0 7`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-navbar bui-navbar-fixed"
  >
    <div
      className="bui-navbar-left"
    >
      <svg
        className="bui-svg-icon bui-navbar-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M16.4 48 50 17.2c1.2-1.1 1.3-3 .2-4.2-1.1-1.2-3-1.3-4.2-.2l-36 33a2.97 2.97 0 0 0 0 4.4l36 33c1.2 1.1 3.1 1 4.2-.2s1-3.1-.2-4.2L16.4 48z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-navbar-title ellipsis"
    >
      标题文案
    </div>
    <div
      className="bui-navbar-right"
    />
  </div>
  <div
    className="bui-navbar-placeholder"
    style={
      Object {
        "height": 0,
      }
    }
  />
</div>
`;
